graphql/schema/schema.graphql

```diff
@@ -175,6 +175,9 @@ type Mutation {
   """Resets the o-counter for a scene to 0. Returns the new value"""
   sceneResetO(id: ID!): Int!
 
+  """Updates resume time"""
```

Comment:
This needs to be documented in detail what happens in this call. For example, it's not clear that the play count is incremented on this call. 

To that end, I suspect that we might want more finer control over the scene count.

Reply:
I meant to update that comment after a few refactors did to that method.

In that last sentence. I assume you mean finner control over the play count?

Reply:
Sorry yes, play count.

Reply:
See my other comments around this.